src/main/java/com/mercadopago/client/order/OrderClient.java

```diff
@@ -110,6 +111,76 @@ public Order get(String id, MPRequestOptions requestOptions) throws MPException,
         result.setResponse(response);
 
         return result;
+    }
+
+    /**
+     * Method responsible for process an order by ID
+     *
+     * @param id orderId
+     * @return order response
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public Order process(String id) throws MPException, MPApiException {
+        return this.process(id, null);
+    }
+
+    /**
+     * Method responsible for process an order by ID with request options
+     *
+     * @param id orderId
+     * @param requestOptions metadata to customize the request
+     * @return order response
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public Order process(String id, MPRequestOptions requestOptions) throws MPException, MPApiException {
+        LOGGER.info("Sending order process request");
+
+        if (id == null || id.isEmpty()) {
+            throw new IllegalArgumentException("Order id cannot be null or empty");
+        }
+
+        String processUrl = String.format(URL_PROCESS, id);
 
+        MPResponse response = send(processUrl, HttpMethod.POST, null, null, requestOptions);
+
+        Order result = Serializer.deserializeFromJson(Order.class, response.getContent());
+        result.setResponse(response);
+
+        return result;
+    }
+
+    /**
+     * Method responsible for delete an order
+     *
+     * @param idOrder orderId
+     * @param idTransaction transactionId
+     * @return order response
+     * @throws MPException an error if the request fails
+     * @throws MPApiException an error if the request fails
+     */
+    public Order delete(String idOrder, String idTransaction ) throws MPException, MPApiException {
+        return this.process(idOrder);
+    }
```

Comment:
Está passando um transactionId e não está utilizando.
Está chamando o process ao invés de delete.
```suggestion
    public Order delete(String idOrder) throws MPException, MPApiException {
        return this.delete(idOrder, null);
    }
```